modules/assignment/src/main/java/com/intuit/wasabi/assignment/AssignmentsModule.java

```diff
@@ -175,6 +180,30 @@ private void bindMetadataCache(final Properties properties) {
         }
     }
 
+    private void bindAssignmentsHourlyAggregation() {
+        //This is the assignment aggregation interval.
+        Integer assignmentAggregationIntervalInMinutes = 60;
```

Comment:
@mooxter Would it make sense to extract & configure the actual value in a properties (assignment.properties?) file, instead of hardcoding it here?